Automattic/wp-calypso
# Thank you card: improve Jetpack set up experience

#### Changes proposed in this Pull Request

* Removes list of features below the card.
* Tweaks dimensions of the buttons.
* Cleans up thank you card styles and makes it very minimal.
* Links the CTA button to `my-plan` if JP >= `6.9-beta`; otherwise it links to `plans`. See: https://github.com/Automattic/jetpack/pull/10556

#### Testing instructions

* Start on `/plans/[site-url]`.
* Purchase a plan.
* Verify the set up progress and thank you card.

#### Before

![image](https://user-images.githubusercontent.com/390760/49153877-e57bfd00-f30e-11e8-92f2-8001cba0ef2f.png)

#### After



![image](https://user-images.githubusercontent.com/390760/49307017-d7bfa680-f4cb-11e8-9f77-872eac6b855a.png)



Fixes #28311
Fixes #25582
